### DISCUSSION TITLE
themes

### DISCUSSION THREAD

**Discussion by RHYTHEM100** [community/community#27711] [Other Feature Feedback, Questions, & Ideas]
how i can remove themes from my page

**Comment by fernandounger** [comment]
> how i can remove themes from my page

When you choose a theme, GitHub adds a line to your repository’s ```_config.yml``` like this:

![image](https://user-images.githubusercontent.com/87918106/181920294-1aed96e0-a571-4cf0-9969-fccdb6c45df4.png)

![image](https://user-images.githubusercontent.com/87918106/181920277-ff551121-2065-4861-9dfb-3777004d15e7.png)

You can delete that line or the entire file if there is nothing else in your ```_config.yml``` and that should remove the theme.

I hope that helps!

**Comment by jfullstackdev** [comment]
also, you can customize the page you want through pure HTML, CSS and JavaScript
